### PR TITLE
Add CUAV C-RTK 9P documentation.

### PR DESCRIPTION
**Pull request by cuhome**
The C-RTK 9P is an RTK receiver developed by CUAV based on the Ublox F9P module.

### CODE REVIEW

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.. _common-cuav-c-rtk-9p-gps:
+
+=============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: Must have the same size of the text.

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+CUAV C-RTK 9P GPS
+=============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: Same, the ===== must have the same length of the title.

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+.. image:: ../../../images/c-rtk-9p/c-rtk-9p.jpg
+	:target: ../images/c-rtk-9p/c-rtk-9p.jpg
+
+.. note::
+
+     C- RTK 9P GPS is supported in Copter-3.6.10 (and higher), Plane 3.9.9 (and higher)
+     
+Key Features{#datasheet}
+============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: same

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+   -  UBX binary
+   -  RTCM version 3.x
+   -  Supply voltage
+   -  –40 °C to +85 °C
+-  **Size**  
+   -  31.5\*48\*12MM
+   -  weight:21g 
+
+Where to buy
+============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: same

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+You can buy it from `CUAV store <https://www.aliexpress.com/item/4000120384761.html>`__.
+
+Connect to Pixhawk
+============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: same

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+.. image:: ../../../images/c-rtk-9p/connect-to-pixhawk.jpg
+	:target: ../images/c-rtk-9p/connect-to-pixhawk.jpg
+	
+Connect to CUAV V5+ Or V5 nano
+============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: same

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+.. image:: ../../../images/c-rtk-9p/c-rtk-9p-connection.jpg
+	:target: ../images/c-rtk-9p/c-rtk-9p-connection.jpg
+	
+.. note::
+
+         Please use V5+ or V5 nano special cable,10 pin connector plugged into GPS&SAFETY PORT, 6 pin connector plugged into UART 4.
+
+More information
+============
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: same

**Comment by brunoolivieri** [comment]
Overall, is that different from other GPS already listed?

**Comment by tridge** [comment]
@brunoolivieri yes, the F9P is very different from the M8 GPS modules we have listed so far. I hope we will have a lot more F9 GPS available soon.
See this article for how much difference it can make:
https://discuss.ardupilot.org/t/comparing-a-ublox-zed-f9p-gps-with-a-m8p-without-external-corrections/43435

**Comment by brunoolivieri** [comment]
Thanks @tridge!

@cuhome Please, could you fix the commented items?

**Comment by auturgy** [comment]
Thanks for reviewing.
This one is waiting on me unfortunately, unless someone else has time. @cuhome is Hongle from CUAV, who is going to be contributing to documentation for them. Intent is to use this PR as a tutorial for him.
We need to fix up the titles, but don’t really need to squash, as that happens on merge for the wiki.

Regards,

James

**Comment by brunoolivieri** [comment]
@auturgy sure!  I cant Merge but I can fiz the titles after that.

Cheers

**Comment by Hwurzburg** [comment]
I can work on it Friday...if content is reviewed and approved, will merge it

**Comment by cuhome** [comment]
@brunoolivieri @Hwurzburg I have modified it, please help me check it out.

File: common/source/docs/common-cuav-c-rtk-9p-gps.rst [hunk 8]
```diff
@@ -0,0 +1,136 @@
+.. _common-cuav-c-rtk-9p-gps:
+
+=============================
```
Comment (brunoolivieri) [inline common/source/docs/common-cuav-c-rtk-9p-gps.rst]: Too long

**Pull request merged**